src/common/selectors.js

```diff
@@ -408,3 +409,13 @@ export const createGetObjectMessages = objectSelector =>
 // const object = getObject(store.getState(), objectId)
 // ...
 export const getObject = createGetObject((_, id) => id)
+
+export const createGetHostMetrics = hostSelector => create(
+  hostSelector,
+  hosts => ({
+    count: size(hosts),
+    cpus: reduce(hosts, (total, host) => total + +host.cpus.cores, 0),
+    memoryTotal: reduce(hosts, (total, host) => total + +host.memory.size, 0),
+    memoryUsage: reduce(hosts, (total, host) => total + +host.memory.usage, 0)
+  })
+)
```

Comment:
Use Dashboard/Overview implementation because:
- it's more efficient, only iterate once over the collection
- it use `createCollectionWrapper()` which returns the previous result if the collection has not really changed